en/README.md

```diff
@@ -2,27 +2,53 @@
 
 [![Releases](https://img.shields.io/github/release/PX4/Firmware.svg)](https://github.com/PX4/Firmware/releases) [![Discuss](https://img.shields.io/badge/discuss-px4-ff69b4.svg)](http://discuss.px4.io/) [![Slack](https://px4-slack.herokuapp.com/badge.svg)](http://slack.px4.io) 
 
-> **Info** Developers only! This guide is under active development and not intended for consumers.
+> **Info** This guide is for primarily for software developers and (new) hardware integrators.
+> To fly, build and modify vehicles using supported hardware see the [PX4 User Guide](https://docs.px4.io/en/).
 
-This guide describes how to work inside the PX4 system architecture. It enables developers to:
+This guide explains how to:
```

Comment:
@LorenzMeier @bkueng This update to the home page is intended to provide better explanation of what you can do with the DevGuide. Some of it is "aspirational" - currently we don't really explain how you can "add new flight modes", but this is where that information should go.